code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -170,6 +172,22 @@
 /// If you wanna try someday feel free, but I can't manage it
 /datum/plane_master_group/popup
 
+/// This is janky as hell but since something changed with CENTER positioning after build 1609 we have to switch to the bandaid LEFT,TOP positioning
+/// using LEFT,TOP at or before 1609 will result in another broken offset for cameras
+#define MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS 1609
+
+/datum/plane_master_group/popup/attach_to(datum/hud/viewing_hud)
+	// If we're about to display this group to a mob who's client is more recent than the last known version with working CENTER, then we need to remake the relays
+	// with the correct screen_loc using the relay override
+	if(viewing_hud.mymob?.client?.byond_build > MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS)
+		for(var/thing in plane_masters)
+			var/atom/movable/screen/plane_master/plane = plane_masters[thing]
+			QDEL_LIST(plane.relays)
```

Comment:
Is it okay to delete this list or maybe simply Cut() it? Will this cause hard deletes? There has been a lot of that going around so not sure

Reply:
I used QDEL_LIST because that's what Destroy used to clean up the relays. Unfortunately there's not like a clear_relays() proc already built in to wipe the slate, so i figured deleting them was the way to go

Reply:
also it wasn't throwing hard del warnings in the server prompt when i would open and close the window

Reply:
Fair enough,  idk much about this code so I'll trust you on this

Reply:
Lemon told me that QDEL_LIST does work but isn't the truly clean way of doing it and to use remove_relay_from for each and I got that working but i'm waiting for him to get back to me on if it looks good enough to get the thumbs up from him

Reply:
ok i think we reached the most succinct and responsible way to regenerate the planes and relays